src/lib/getMimeTypeIcon.spec.js

```
@@ -0,0 +1,18 @@
+import IconFiles from 'cozy-ui/transpiled/react/Icons/FileTypeFiles'
+
+import ExcalidrawIcon from '@/assets/icons/icon-excalidraw.svg'
+import getMimeTypeIcon from '@/lib/getMimeTypeIcon'
+
+describe('getMimeTypeIcon', () => {
+  it('should return the excalidraw icon for .excalidraw files', () => {
+    const icon = getMimeTypeIcon(false, 'My drawing.excalidraw', 'application/json')
```

ESLint

[error] Replace `false,·'My·drawing.excalidraw',·'application/json'` with `⏎······false,⏎······'My·drawing.excalidraw',⏎······'application/json'⏎····`

(prettier/prettier)